packages/datetime/src/_daterangeinput.scss

```diff
@@ -8,9 +8,38 @@
 /*
 Date range input
 
+The `DateRangeInput` component is a [control group](#components.forms.control-group) composed of two
+[input groups](#components.forms.input-group). It shows a
+[`DateRangePicker`](#components.datetime.daterangepicker) in a [`Popover`](#components.popover) on focus.
+
+Use the `onChange` function to listen for changes to the selected date. Use `onError` to listen for
+invalid entered dates or date ranges.
+
+You can control the selected date by setting the `value` prop, or use the component in uncontrolled
+mode and specify an initial date by setting `defaultValue`.
```

Comment:
this and paragraph above belong in JavaScript API section below as they describe how to use the React component itself